dotenv/README.md

```diff
@@ -2,113 +2,76 @@
 
 ## Usage
 
-Setup a `.env` file in the root of your project.
+Setup a `.env` file in your project.
 
 ```sh
 # .env
 GREETING=hello world
 ```
 
-Then import the configuration using the `config` function.
+Then import the configuration using the `parse` functions.
 
 ```ts
 // app.ts
-import { config } from "https://deno.land/std@$STD_VERSION/dotenv/mod.ts";
+import { parse } from "https://deno.land/std@$STD_VERSION/dotenv/mod.ts";
+const source = await Deno.readTextFile("./path/to/.env");
+const { env } = parse(source);
 
-console.log(await config());
+console.log(env);
 ```
 
 Then run your app.
 
 ```
-> deno run --allow-read app.ts
+> deno app.ts
 { GREETING: "hello world" }
 ```
 
 ### Options
 
-- `path?: string`: Optional path to `.env` file. Defaults to `./.env`.
-- `export?: boolean`: Set to `true` to export all `.env` variables to the
-  current processes environment. Variables are then accessable via
-  `Deno.env.get(<key>)`. Defaults to `false`.
-- `safe?: boolean`: Set to `true` to ensure that all necessary environment
-  variables are defined after reading from `.env`. It will read `.env.example`
-  to get the list of needed variables.
-- `example?: string`: Optional path to `.env.example` file. Defaults to
-  `./.env.example`.
+- `example?: DotEnvObject`: example object that the parsed data will be verified
+  against.
 - `allowEmptyValues?: boolean`: Set to `true` to allow required env variables to
   be empty. Otherwise it will throw an error if any variable is empty. Defaults
   to `false`.
-- `defaults?: string`: Optional path to `.env.defaults` file which defaults to
-  `./.env.defaults`.
 
-### Auto loading
+## Auto loading
 
-`load.ts` automatically loads the local `.env` file on import and exports it to
-the process environment:
+`load` and `loadSync` automatically loads the local `.env` file and exports it
+to the passed process environment object:
 
 ```sh
 # .env
 GREETING=hello world
 ```
 
+### load
+
 ```ts
 // app.ts
-import "https://deno.land/std@$STD_VERSION/dotenv/load.ts";
-
+import { load } from "https://deno.land/std@$STD_VERSION/dotenv/load.ts";
+await load(Deno.env);
```

Comment:
I'm not in favor of this change. In my view most of use cases of this module is this pattern. So this adds some boilerplates to the main use case. This seems making this library a little inconvenient

Reply:
Do you mean adding `Deno.env` as an argument and/or the function call?

Reply:
I mean the function call looks redundant. People should be interested only in that use case. So it's better to do that automatically in my view.

Reply:
In effort toward preserving existing behavior: How about moving everything in `load.ts` to a new module (e.g. `loader.ts`), then import the `load` function into `load.ts` and invoke it at the top level?